app.js

```diff
@@ -1,4 +1,5 @@
 const { App, LogLevel, Assistant } = require('@slack/bolt');
+
```

Comment:
👾  question: Am curious if this is recommended import ordering? IIRC `@` precedes letters and imported packages appear before project imports, but am not sure of the space?